sources/packages/web/src/services/http/BypassedRestrictionApi.ts

```diff
@@ -0,0 +1,22 @@
+import HttpBaseClient from "@/services/http/common/HttpBaseClient";
+import { BypassedRestrictionSummaryAPIOutDTO } from "@/services/http/dto";
+
+/**
+ * Http API client for Bypassed Restrictions.
+ */
+export class BypassedRestrictionApi extends HttpBaseClient {
+  /**
+   * API client to call the application bypassed restrictions rest API.
```

Comment:
Minor, we avoid technical terms like rest API or API in comments and try to keep it more business oriented and user friendly

Reply:
I didn't remove this one because it's minor and most of other web Api files display the comments for the first method. Same for the backend APi as well. Please let me know if this is mandatory and I'll make the change.

Reply:
Considering it not a blocker.